apps/condo/domains/property/components/PropertyForm/UpdatePropertyForm.tsx

```diff
@@ -7,27 +7,34 @@ import { useOrganization } from '@open-condo/next/organization'
 
 import { Button } from '@condo/domains/common/components/Button'
 import { Loader } from '@condo/domains/common/components/Loader'
+import { isSafeUrl } from '@condo/domains/common/utils/url.utils'
 import { Property } from '@condo/domains/property/utils/clientSchema'
 
 import BasePropertyForm from '../BasePropertyForm'
 
 interface IUpdatePropertyForm {
     id: string
+    next?: string,
```

Comment:
Why are you passing this value through several components? Why not just get it from URL?